src/test/java/org/prebid/server/bidder/undertone/UndertoneBidderTest.java

```diff
@@ -96,20 +99,56 @@ public void errorOnInvalidImps() {
 
     @Test
     public void testValidBidRequest() {
-        final BidRequest bidRequest = givenBidRequest(bidRequestBuilder -> bidRequestBuilder.site(Site.builder()
-                        .id("site-id")
-                        .build()),
-                impBuilder -> impBuilder
-                        .id("imp-id")
-                        .banner(Banner.builder()
-                                .id("banner-id")
-                                .w(300)
-                                .h(600)
-                                .build())
-                        .ext(mapper.valueToTree(ExtPrebid.of(null, ExtImpUndertone.of(1234, 12345)))));
+        final Site site = Site.builder()
+                .id("site-id")
+                .build();
+
+        final String bidReqId = "req-id";
+        final int publisherId = 1234;
+        final int placementId = 12345;
```

Comment:
We prefer not to use such variables in tests, but set values without previous variables creation
Like
``` java
ExtImpUndertone.of(1234, 12345)
final Publisher expectedPublisher = Publisher.builder()
                .id("1234")
                .build();
```